opre_ops/ops_site/migrations/0007_canamount_caninfo.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 3.2.6 on 2021-09-17 21:45
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ops_site', '0006_alter_person_division'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='CANInfo',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('number', models.CharField(max_length=30)),
+                ('description', models.CharField(max_length=100)),
+                ('nickname', models.CharField(max_length=30)),
+                ('purpose', models.TextField(null=True)),
+                ('arrangement_type', models.CharField(choices=[('1', 'OPRE Appropriation'), ('2', 'Cost Share'), ('3', 'IAA'), ('4', 'IDDA'), ('5', 'MOU')], max_length=30)),
+                ('authorizer', models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, related_name='authorizer', to='ops_site.agency')),
+                ('source', models.ManyToManyField(to='ops_site.Agency')),
+            ],
+            options={
+                'verbose_name_plural': 'CANs',
+            },
+        ),
+        migrations.CreateModel(
+            name='CANAmount',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('fiscal_year', models.CharField(max_length=5)),
+                ('amount_available', models.DecimalField(decimal_places=2, max_digits=10)),
+                ('amount_budgeted', models.DecimalField(decimal_places=2, max_digits=10)),
+                ('additional_amount_anticipated', models.DecimalField(decimal_places=2, max_digits=10)),
+                ('notes', models.TextField()),
```

Comment:
[_issue, blocking_]: Similar to the issue about the `purpose` field above. Suggest that we use `blank=True` here. This should resolve an issue where CAN amounts fail to save in Django Admin.

https://docs.djangoproject.com/en/3.2/ref/models/fields/#null
https://docs.djangoproject.com/en/3.2/ref/models/fields/#blank